relationships/candidates/KUBERNETESCLUSTER.yml

```diff
@@ -4,8 +4,10 @@ lookups:
     - domain: INFRA
       type: KUBERNETESCLUSTER
     tags:
-      matchingMode: ANY
+      matchingMode: FIRST
```

Comment:
We are changing the matching mode and it is not present on the description of the PR. 

Could we please include the reasoning for this change on the PR description?